setup.cfg

```diff
@@ -0,0 +1,51 @@
+[metadata]
+name = broadbean
+maintainer = QCoDeS Core Developers
+maintainer_email = qcodes-support@microsoft.com
+description = Package for easily generating and manipulating signal 
+                pulses. Developed for use with qubits in the quantum 
+                computing labs of Copenhagen, Delft, and Sydney, but 
+                should be generally useable.
+keywords = Pulsebuilding signal processing arbitrary waveforms
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/QCoDeS/broadbean
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering
+license = MIT
+
+[options.package_data]
+broadbean =
+    py.typed
+
+[options]
+include_package_data = True
+packages=find:
+python_requires = >=3.7
+# We might as well require what we know will work
+# although older numpy and matplotlib version will probably work too
+install_requires = 
```

Comment:
This comment is very outdated. That version of numpy is unlikely to work with python 3.7

Reply:
thanks I'll remove the comment, shall we remove 3.7 support in a follow up pr?